addons/purchase/models/stock.py

```diff
@@ -60,7 +60,7 @@ def get_price_unit(self):
     def copy(self, default=None):
         self.ensure_one()
         default = default or {}
-        if not default.get('split_from'):
+        if not default.get('split_from') and not default.get('origin_returned_move_id'):
```

Comment:
Could you explain why we do this ? I'm not familiar with the use case of copying stock moves.

Could be nice to also update the comment that follows.